benchmarks/different_model_options.py

```diff
@@ -37,17 +37,6 @@ class SolveModel:
     t_interp: np.ndarray | None
 
     def solve_setup(self, parameter, model_, option, value, solver_class):
-        import importlib
-
-        idaklu_spec = importlib.util.find_spec("pybamm.solvers.idaklu")
-        if idaklu_spec is not None:
-            try:
-                idaklu = importlib.util.module_from_spec(idaklu_spec)
-                idaklu_spec.loader.exec_module(idaklu)
-            except ImportError as e:  # pragma: no cover
-                print("XXXXX cannot find klu", e)
-                idaklu_spec = None
-
```

Comment:
This will make the IDAKLU solver non-optional, is this also intended here? This would cause non-supported platforms such as Linux arm64 (which we provide Docker images for) try to compile `pybammsolvers` from the sdist, which is going to be broken until https://github.com/pybamm-team/pybammsolvers/pull/7 is merged.

 I think it's better to split both tasks (separating the solver, and making the solver the default) in two separate PRs.

Reply:
Yeah changing the default solver is intended to be a separate task.

I thought the linux Arm64 stuff was already fixed. I go back to working on that

Reply:
Sure. The last time I worked on Linux arm64, cross-compilation wasn't helpful with SuiteSparse's symbols turning up missing, and CasADi faced https://github.com/dockcross/dockcross/issues/780 when I tried compiling under QEMU. But maybe the situation has improved by a lot now...

Reply:
I am working on getting the scikit build up and running again while also looking at the conda-forge recipe. Hopefully those will resolve these issues